LondonBritishLibrary/orient/BLorient753.xml

```diff
@@ -1,17 +1,14 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 schematypens="http://relaxng.org/ns/structure/1.0"
-?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
    schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BLorient753" type="mss">
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Qalemǝnṭos, Ṭǝnta Haymānot, The religious Romance (The History of Baralām wa-Yǝwāsǝf)
-                    
-                    
-                    
-                    
+                <title>Qalemǝnṭos, Maṣḥafa ṭǝnta hāymānot, Baralām wa-Yǝwāsǝf
                 </title>
                 <editor key="SG"/>
```

Comment:
I think in all those cases, SG should only be mentioned in the change element as he only created stubs, without the proper description